README.md

```diff
@@ -21,7 +29,7 @@ import ShopifyFormat from "@shopify/i18next-shopify";
 i18next.use(ShopifyFormat).init(i18nextOptions);
 ```
 
-## More complete sample
+## Example
```

Comment:
I'm not sure where, but it would be good to put somewhere in the README that this plugin supports interpolation variables with single and double curly braces